packs/data/journals.db/gm-screen.json

```diff
@@ -35,8 +35,9 @@
-                "format": 1
+                "content": "<p><span style=\"font-family:Signika, sans-serif\">Below you will find list of shortcuts to subjects that are useful for to players, grouped by theme<br /></span><strong><br />Reference Tables</strong> <strong>Actions &amp; Skills</strong></p><table class=\"pf2-table\"><tbody><tr><td style=\"width:25%\">@UUID[.8qYJAx4O97sSxbUF]{Basic Actions}</td><td style=\"width:25%\">@UUID[.duqPy1VMQYNJrw7q]{Conditions}</td><td style=\"width:25%\">@UUID[.mmfWcV3Iyql5nzTo]{Counteract}</td><td style=\"width:25%\">@UUID[.Z2xXPHkxGLFTXzdI]{Creature Identification}</td></tr><tr><td>@UUID[.Edn6qQNVcEVpkhPl]{Detecting Creatures}</td><td>@UUID[.aBALUjLyOqXKlhrP]{Earn Income}</td><td>@UUID[.zB1mqE9IeyfyQDnn]{Exploration Activities}</td><td>@UUID[.XCxK6OGFY0KvJPo3]{Learn a Spell}</td></tr><tr><td>@UUID[.7jQ2x83KUFD8Wj6w]{Light Effects}</td><td>@UUID[.SiMXtLf4YQeigzIE]{Senses}</td><td>@UUID[.uWJfFysCcoU1o2Hs]{Skill Actions}</td><td></td></tr></tbody></table><p><strong>Death &amp; Recovery</strong> <strong>Terrain &amp; Environment</strong></p><table class=\"pf2-table\"><tbody><tr><td style=\"width:25%\">@UUID[.tJllU1E8g7W1QxYe]{Cover}</td><td style=\"width:25%\">@UUID[.uxdPLsxY8fNrenDR]{Death and Dying}</td><td style=\"width:25%\">@UUID[.7FsXXkrOUXHtROnq]{Falling}</td><td style=\"width:25%\">@UUID[.pEjd65isUfXKSB18]{Hero Points}</td></tr><tr><td>@UUID[.I6lHRH3mWjfvhJrR]{Resting}</td><td>@UUID[.G1Rje9eYCFEByzjg]{Special Battles}</td><td>@UUID[.cjD6YVPQpmYiOPdC]{Terrain in Encounters}</td><td>@UUID[.YVLMgNxXTUQuDJgp]{Travel Speed}</td></tr><tr><td>@UUID[.kL8yx8vz9FMCIYC1]{Treat Wounds}</td><td></td><td></td><td></td></tr></tbody></table><p><strong>Equipment</strong></p><table class=\"pf2-table\"><tbody><tr><td style=\"width:33%\">@UUID[.6BXJUVvwvzljaCFT]{Armor Traits}</td><td style=\"width:34%\">@UUID[.VR9UdfhZphZGZdsx]{Bulk Conversions for Different Sizes}</td><td style=\"width:33%\">@UUID[.PVrMSF93KIejBTMm]{Critical Specialization Effects}</td></tr><tr><td>@UUID[.act12TfHpnKx6ZTu]{Different Size Items}</td><td>@UUID[.dnbUao9yk8Bs99eT]{Weapon Traits}</td><td></td></tr></tbody></table>",
```

Comment:
`<span style=\"font-family:Signika, sans-serif\">`

We want to reduce the number of inline styles we have. With tables it's forced in by Prose Mirror/TinyMCE (Foundry's text editors), and there's not much we can do about it. But other text we should remove these span tags entirely (plus the closing `</span>` they pair with).